### ISSUE TITLE
SMC: 2FA Message not Received for Partner

### ISSUE THREAD

**Issue by iph-97** [bloom-housing/bloom#3587]
A property manager with the account ryan.dierks@greystar.com is reporting that every time they try to log in with 2FA, they do not receive a text or email. They manage a handful of properties in SMC. Exygy staff seem to be receiving emails for 2FA just fine. This is a medium priority request :large_yellow_circle:.

**Comment by sarahlazarich** [comment]
We are waiting on this one, context [here](https://exygy.slack.com/archives/C02G1S19QA2/p1691535588685859)

**Comment by ludtkemorgan** [comment]
Failing emails appeared to have been a temporary issue and is no longer happening. The texting issue is a bigger issue with our current setup in Twilio. The number we have was unregistered and no campaign associated to it. This means that the messages are throttled and it will get worse as time goes on. Unfortunately we were not informed of this needing to be resolved until the errors started happening.

We have registered the number and applied for a campaign. However, it takes several weeks to get approved and we are still waiting on that. In the meantime we have disabled the "Phone" option in HBA partner site which will force people to use email as their MFA option. I will close this issue when the Twilio issue is resolved and we can re-enable it.
